### PR TITLE
Zth iteration4

### PR DESCRIPTION
**Pull request by zachholcomb**
I completed the biggest bust method
I used  a sub to test the method, had to add mocha/minitest to the test_helper because my test_helper didn't hapen
Hope it doesn't cause too many issues

### CODE REVIEW

File: lib/season_stat.rb [hunk 1]
```diff
@@ -58,7 +57,7 @@ def get_team_info(season)
          season_win_percent: team_win_percentage(team.team_id, 'Regular Season', season),
          postseason_win_percent: team_win_percentage(team.team_id, 'Postseason', season)
 }
-      @team_info = team_hash
+      team_hash
```
Comment (zachholcomb) [inline lib/season_stat.rb:60]: refactored @team_info out of season class, it seemed unnecessary to store a hash with season variables when i could just call the method and take the return value

File: lib/season_stat.rb [hunk 2]
```diff
@@ -92,9 +91,10 @@ def team_win_percentage(team_id, game_type, season)
     ((total_wins / total_games) * 100).round(2)
   end
 
-  def get_regular_percents(game_type)
```
Comment (zachholcomb) [inline lib/season_stat.rb:95]: refactored this helper method out of season too, it was redundant and I was accomplishing this in the get_team_info method

File: lib/season_stat.rb (hunk 1 above)
Comment (elguapogordo) [reply lib/season_stat.rb:60]: i agree here!

File: lib/season_stat.rb [hunk 3]
```diff
@@ -92,9 +91,10 @@ def team_win_percentage(team_id, game_type, season)
...
-    @team_info.map do |team_info, team|
-      team_info[team] = team_win_percentage(team, game_type)
+  def biggest_bust(season)
+    team_bust = get_team_info(season).max_by do |team_id, team_info|
+      (team_info[:season_win_percent] - team_info[:postseason_win_percent])
     end
+    team_bust[1][:team_name]
   end
```
Comment (zachholcomb) [inline lib/season_stat.rb:99]: max_by returned an array so I had to pull the values out here that I needed

File: test/season_stat_test.rb [hunk 4]
```diff
@@ -83,9 +83,28 @@ def test_it_can_calculate_team_win_percentage
...
+            :postseason_win_percent => 70.00
+          },
+      4 => {:team_name=> "Avalanche",
+            :season_win_percent => 50.00,
+            :postseason_win_percent => 25.00
+            }
+    }
+    season = mock('testseason')
+    @season_stat.stubs(:get_team_info).returns(team_info)
+    assert_equal "The Bunnies", @season_stat.biggest_bust(season)
```
Comment (zachholcomb) [inline test/season_stat_test.rb:107]: here is the test, used a mock and a stub for this test

**Pull request merged**